Frontend/ui-driver/src/Components/BottomNavBar/Controller.purs

```diff
@@ -16,63 +16,69 @@
-      text: "Home"
-    },
-    {
-      activeIcon: if (getMerchant FunctionCall == NAMMAYATRI) then "ny_ic_rides_active," <> (getCommonAssetStoreLink FunctionCall) <> "ny_ic_rides_active.png" else "ny_ic_cab_active,https://assets.juspay.in/beckn/merchantcommon/images/ny_ic_cab_active.png",
-      defaultIcon: if (getMerchant FunctionCall == NAMMAYATRI) then "ny_ic_rides_inactive," <> (getCommonAssetStoreLink FunctionCall) <> "ny_ic_rides_inactive.png" else "ny_ic_cab_inactive,https://assets.juspay.in/beckn/merchantcommon/images/ny_ic_cab_inactive.png",
-      text: "Rides"
-    }] <>
-    (if (getMerchant FunctionCall == NAMMAYATRI) then [
-    {
-      activeIcon: "ny_ic_join_active,https://assets.juspay.in/nammayatri/images/driver/ic_profile_active.png",
-      defaultIcon: "ny_ic_join_inactive,https://assets.juspay.in/nammayatri/images/driver/ic_profile_inactive.png",
-      text: "Join"
-    },
-    {
-      activeIcon: "ic_referral_active," <> (getAssetStoreLink FunctionCall) <> "ic_referral_active.png",
-      defaultIcon: if (getValueToLocalNativeStore REFERRAL_ACTIVATED) == "true" then  "ny_ic_contest_alert," <> (getCommonAssetStoreLink FunctionCall) <> "ny_ic_contest_alert.png" else "ic_referral_inactive," <> (getCommonAssetStoreLink FunctionCall) <> "ic_referral_inactive.png",
-      text: "Rankings"
-    }] else [
-    {
-      activeIcon: "ic_referral_active," <> (getAssetStoreLink FunctionCall) <> "ic_referral_active.png",
-      defaultIcon: if (getValueToLocalNativeStore REFERRAL_ACTIVATED) == "true" then  "ny_ic_contest_alert," <> (getCommonAssetStoreLink FunctionCall) <> "ny_ic_contest_alert.png" else "ic_referral_inactive," <> (getCommonAssetStoreLink FunctionCall) <> "ic_referral_inactive.png",
-      text: "Rankings"
-    }
-    ]) <> 
-    [{
-      activeIcon: "ny_ic_alerts_active",
-      defaultIcon: "ny_ic_alerts_inactive," <> (getCommonAssetStoreLink FunctionCall) <> "ny_ic_alerts_inactive.png",
-      text: "Alert"
-    }
-  ]
-}
+navData :: ScreenNames.ScreenName -> BottomNavConfig -> BottomNavBarState
+navData screenName bottomNavConfig = do
+  let showNewBannerOnSubscription = (Maybe.fromMaybe 0 $ fromString $ getValueToLocalNativeStore TIMES_OPENED_NEW_SUBSCRIPTION) < 3
+      navdata = [
+        {
+          activeIcon: "ny_ic_home_active," <> (getCommonAssetStoreLink FunctionCall) <> "ny_ic_home_active.png",
+          defaultIcon: "ny_ic_home_inactive," <> (getCommonAssetStoreLink FunctionCall) <> "ny_ic_home_inactive.png",
+          text: "Home",
+          showNewBanner : bottomNavConfig.home.showNew,
+          isVisible : bottomNavConfig.home.isVisible,
+          screenName : ScreenNames.HOME_SCREEN
+        },
+        {
+          activeIcon: if (getMerchant FunctionCall == NAMMAYATRI) then "ny_ic_rides_active," <> (getCommonAssetStoreLink FunctionCall) <> "ny_ic_rides_active.png" else "ny_ic_cab_active,https://assets.juspay.in/beckn/merchantcommon/images/ny_ic_cab_active.png",
```

Comment:
We can make this activeIcon also configurable, So that we don't need to check on merchant. @witcher-shailesh